packages/ui-app/src/types.d.ts

```diff
@@ -25,3 +25,13 @@ export type BaseContext = {
     }
   }
 };
+
+export type UnlockI18n = {
+  key: string,
+  value: any
+};
+
+export type InputErrorMessage = {
```

Comment:
Ok. I did comment on this in the previous iteration - the error needs to be defined as `React.ReactNode` as per the previous comment. NOT this.

Can we please get it right. The translation happens on the calling component and calling component only.